basiclti/basiclti-impl/src/java/org/sakaiproject/lti/impl/DBLTIService.java

```diff
@@ -461,20 +463,48 @@ public Object updateContentDao(Long key, Object newProps, String siteId,
 	public List<Map<String, Object>> getContentsDao(String search, String order, int first,
 			int last, String siteId, boolean isAdminRole) {
 
-		if ( order != null ) {
-			order = foorm.orderCheck(order, "lti_content", LTIService.CONTENT_MODEL);
-			if ( order == null ) {
-				throw new IllegalArgumentException("order must be [table.]field [asc|desc]");
-			}
-		}
-
-		List<Map<String, Object>> contents = getThingsDao("lti_content",
-				LTIService.CONTENT_MODEL, null, null, search, null, order, first, last, siteId, isAdminRole);
-		for (Map<String, Object> content : contents) {
-			content.put("launch_url", getContentLaunch(content));
-		}
-		return contents;
+        String extraSelect = "SAKAI_SITE.TITLE AS SITE_TITLE, ssp1.VALUE AS SITE_CONTACT_NAME, ssp2.VALUE AS SITE_CONTACT_EMAIL";
```

Comment:
Doing joins in the DB and not going through the services in Sakai is normally frowned upon as it means that any caching or refactoring of the original tables becomes much harder. It also means that security checks can be unintentionally bypassed.